crates/forge/src/lib.rs

```diff
@@ -50,8 +50,10 @@ const MINIMAL_SCARB_VERSION_FOR_SIERRA_GAS: Version = Version::new(2, 10, 0);
 Use -h for short descriptions and --help for more details.
 
 {before-help}{usage-heading} {usage}
+{subcommands}
+  \x1b[1mclean cache\x1b[0m         Clean Forge cache directory 
 
-{all-args}{after-help}
+{after-help}
```

Comment:
That's not needed, `clean` is already included in `--help` output, please revert these.